actions/admin/stats.go

```diff
@@ -8,8 +8,15 @@ import (
 )
 
 // status will return the status of the admin login
```

Comment:
Not your fault. But could you please change "status" to "stats" in the function description